test/acceptance/tools/cygnus.py

```diff
@@ -66,6 +68,8 @@
 CKAN_SINK        = u'ckan'
```

Comment:
Why do you assign a constant value to a variable value instead of using it directly?